packages/pangea-sdk/pangea/asyncio/request.py

```diff
@@ -143,13 +140,13 @@ async def put_presigned_url(self, url: str, files: List[Tuple]):
         resp = await self._http_put(url=url, files=files)
         self.logger.debug(
             json.dumps(
-                {"service": self.service, "action": "put presigned", "url": url, "response": resp.text},
+                {"service": self.service, "action": "put presigned", "url": url, "response": await resp.text()},
                 default=default_encoder,
             )
         )
 
-        if resp.status_code < 200 or resp.status_code >= 300:
```

Comment:
whats this change from status_code to status?

Reply:
It was a bug. asyncio request has `status`